### PR TITLE
Fix broken command-line options

### PR DESCRIPTION
**Pull request by jzuhone**
## PR Summary

This PR fixes various broken command-line options. 

Fixes parts of #3360.

### CODE REVIEW

File: yt/visualization/mapserver/pannable_map.py [hunk 1]
```diff
@@ -134,20 +135,19 @@ def static(self, path):
...
         # Add fluid fields (only gas for now)
         for ftype in self.ds.fluid_types:
             d[ftype] = []
             for f in self.ds.derived_field_list:
                 if f[0] != ftype:
                     continue
-
+                # Discard fields which need ghost zones for now
+                df = self.ds.field_info[f]
+                if any([isinstance(v, ValidateSpatial) for v in df.validators]):
```
Comment (cphyc) [inline yt/visualization/mapserver/pannable_map.py:146]: Note that you can also write to spare two chars
```suggestion
                if any(isinstance(v, ValidateSpatial) for v in df.validators):
```
I am absolutely fine with the current line, so feel free to just close and ignore this snippet.

File: yt/utilities/command_line.py [hunk 2]
```diff
@@ -1477,30 +1477,30 @@ def __call__(self, args):
...
-                print(
-                    f"Maximum {args.field}: {vals['min'][0]:0.5e} at {vals['min'][1]}"
-                )
-            if args.min:
-                vals["max"] = ds.find_min(args.field)
-                print(
-                    f"Minimum {args.field}: {vals['max'][0]:0.5e} at {vals['max'][1]}"
-                )
+        field = ds._get_field_info(args.field)
+        if args.max:
```
Comment (cphyc) [inline yt/utilities/command_line.py:1481]: It looks like the logic is broken here, isn't it?

**Review by cphyc — APPROVED** [review]
Looks good to me!

File: yt/utilities/command_line.py (hunk 2 above)
Comment (jzuhone) [reply yt/utilities/command_line.py:1359]: Sorry, could you elaborate? Worked for me after I made this change.

Comment (neutrinoceros) [reply yt/utilities/command_line.py:1359]: I'm confused as well. For instance the line after this one
```python
vals["min"] = ds.find_max(field)
```
_looks_ broken. Maybe it's not, but in any case I don't understand it as it is

Comment (jzuhone) [reply yt/utilities/command_line.py:1359]: Ah dang--I didn't look at it carefully enough. Yeah, I'll fix it.

Comment (neutrinoceros) [reply yt/utilities/command_line.py:1359]: thanks !

**Review by neutrinoceros — APPROVED** [review]

**Comment by jzuhone** [comment]
pre-commit.ci autofix

File: yt/visualization/mapserver/pannable_map.py (hunk 1 above)
Comment (neutrinoceros) [reply yt/visualization/mapserver/pannable_map.py]: Puypgrade (via pre-commit) is supposed to handle this automatically

**Comment by jzuhone** [comment]
Ok, this is ready to go now

**Pull request merged**